tasks.py

```diff
@@ -499,6 +502,19 @@ def regenerate_package_mode(c, debug=False):
 
     _run_autorest(cmds, debug=debug)
 
+@task
```

Comment:
I would really prefer if we don't keep adding specification readmes across the repos, because then we have to look in multiple locations. I totally get why you needed them for package mode, but instead I think you should add swaggers to the [testserver](https://github.com/Azure/autorest.testserver/tree/main/swagger) and just pass in the names of the swaggers you added.

This way you have cross language impact, and are also testing the more common use case of adding the security configurations to swagger.

There's also less code clutter with adding all of these different readmes, and by default you already generate these swaggers with our three different generators.

Reply:
once you've added the swaggers to the testserver, can you also try generating some of the files in `azure-arm` too (i.e. just include the swagger names in the azure arm section of tasks swaggers)? That way we can make sure the correct overriding is happening with flag `--azure-arm`

Reply:
Good point. Let me have a try. And I think we could also keep the task since it could help check whether security flag in readme works.

Reply:
we don't have to add a whole new readme though, if you really wanted to we could add flags in `tasks.py`. However, I honestly think it clutters up the repo a lot, and would much rather we just use the preferred way of having a swagger in the testserver, so all languages get the benefits

Reply:
Have added `security.ts` in https://github.com/Azure/autorest.testserver/pull/368 add related test: https://github.com/Azure/autorest.python/pull/1161#discussion_r824429304. The test pass in local environment. But I am not sure how to test aad with testserver. If we need to simulate the authentication process, it is a little complicated and I am not very familiar with that. So maybe we could add it later

Reply:
yeah i think that's fair. I'm going to wait to merge the testserver PR until this PR is ready to merge, otherwise we get issues where we don't have coverage, but hopefully you're not blocked on your side